src/binaryen-c.h

```diff
@@ -2960,6 +2994,18 @@ BINARYEN_API void BinaryenSetPassArgument(const char* name, const char* value);
 // Applies to all modules, globally.
 BINARYEN_API void BinaryenClearPassArguments();
 
+// Test whether a pass is in the set of passes to skip.
+// Applies to all modules, globally.
+BINARYEN_API bool BinaryenTestPassToSkip(const char* pass);
```

Comment:
We use "has" elsewhere for stuff like this, e.g. `BinaryenFunctionHasLocalName`, so perhaps
```suggestion
BINARYEN_API bool BinaryenHasPassToSkip(const char* pass);
```

Reply:
I have renamed the functions.